ui/src/views/Tasks/TaskLog/index.jsx

```diff
@@ -43,8 +31,8 @@ export default class TaskLog extends Component {
         <Log
           url={url}
           stream={stream}
-          GoToLineButtonProps={{ className: classes.miniFab }}
-          FollowLogButtonProps={{ className: classes.followButton }}
+          GoToLineButtonProps={{ size: 'small' }}
+          FollowLogButtonProps={{ size: 'small' }}
```

Comment:
Instead of specifying it here, we could probably just do it in the Log component directly:

```diff
diff --git a/ui/src/components/Log/index.jsx b/ui/src/components/Log/index.jsx
index 10a124a8c..a7ab08ef1 100644
--- a/ui/src/components/Log/index.jsx
+++ b/ui/src/components/Log/index.jsx
@@ -280,10 +280,12 @@ export default class Log extends Component {
             {rawLogButton}
             <div className={classes.logActions}>
               <GoToLineButton
+                size="small"
                 onLineNumberChange={this.handleLineNumberChange}
                 {...GoToLineButtonProps}
               />
               <Button
+                size="small"
                 spanProps={{
                   className:
                     FollowLogButtonProps && FollowLogButtonProps.className,
diff --git a/ui/src/views/Tasks/TaskLog/index.jsx b/ui/src/views/Tasks/TaskLog/index.jsx
index c42423e46..048c12d82 100644
--- a/ui/src/views/Tasks/TaskLog/index.jsx
+++ b/ui/src/views/Tasks/TaskLog/index.jsx
@@ -31,8 +31,6 @@ export default class TaskLog extends Component {
         <Log
           url={url}
           stream={stream}
-          GoToLineButtonProps={{ size: 'small' }}
-          FollowLogButtonProps={{ size: 'small' }}
           RawLogButtonProps={{ className: classes.rawLogButton }}
           actions={
             <Button
```

Reply:
Good idea, done!